[2023-04-23 09:33:32,669] numexpr.utils - INFO - NumExpr defaulting to 8 threads.
[2023-04-23 09:33:34,329] root - INFO - Training Pipeline Config: TrainingPipelineConfig(artifact_dir='C:\\Users\\Sumeet Maheshwari\\Desktop\\end to end project\\credit_score_prediction\\credit_score_classification\\Credit_score_Classification\\artifact')
[2023-04-23 09:33:34,329] root - INFO - Data Ingestion Config : DataIngestionConfig(dataset_download_url='https://github.com/sumeet0701/credit_score_classification/blob/main/dataset/dataset_clean.zip?raw=true', tgz_download_dir='C:\\Users\\Sumeet Maheshwari\\Desktop\\end to end project\\credit_score_prediction\\credit_score_classification\\Credit_score_Classification\\artifact\\data_ingestion\\2023-04-23-09-33-32\\tgz_data', raw_data_dir='C:\\Users\\Sumeet Maheshwari\\Desktop\\end to end project\\credit_score_prediction\\credit_score_classification\\Credit_score_Classification\\artifact\\data_ingestion\\2023-04-23-09-33-32\\raw_data', ingested_train_dir='C:\\Users\\Sumeet Maheshwari\\Desktop\\end to end project\\credit_score_prediction\\credit_score_classification\\Credit_score_Classification\\artifact\\data_ingestion\\2023-04-23-09-33-32\\ingested_data\\train', ingested_test_dir='C:\\Users\\Sumeet Maheshwari\\Desktop\\end to end project\\credit_score_prediction\\credit_score_classification\\Credit_score_Classification\\artifact\\data_ingestion\\2023-04-23-09-33-32\\ingested_data\\test') 
[2023-04-23 09:33:34,331] root - INFO - 
******************** Data Ingestion log started ********************

[2023-04-23 09:33:34,334] root - INFO - Connection with DB created successfully!!!
[2023-04-23 09:33:34,335] root - INFO - Downloading file from: [https://github.com/sumeet0701/credit_score_classification/blob/main/dataset/dataset_clean.zip?raw=true] into : [C:\Users\Sumeet Maheshwari\Desktop\end to end project\credit_score_prediction\credit_score_classification\Credit_score_Classification\artifact\data_ingestion\2023-04-23-09-33-32\tgz_data\rental_bike_data.zip]
[2023-04-23 09:33:39,469] root - INFO - File: [C:\Users\Sumeet Maheshwari\Desktop\end to end project\credit_score_prediction\credit_score_classification\Credit_score_Classification\artifact\data_ingestion\2023-04-23-09-33-32\tgz_data\rental_bike_data.zip] has been downloaded successfully
[2023-04-23 09:33:39,471] root - INFO - Extracting zipped file : [C:\Users\Sumeet Maheshwari\Desktop\end to end project\credit_score_prediction\credit_score_classification\Credit_score_Classification\artifact\data_ingestion\2023-04-23-09-33-32\tgz_data\rental_bike_data.zip] into dir: [C:\Users\Sumeet Maheshwari\Desktop\end to end project\credit_score_prediction\credit_score_classification\Credit_score_Classification\artifact\data_ingestion\2023-04-23-09-33-32\raw_data]
[2023-04-23 09:33:39,561] root - INFO - Extraction completed successfully
[2023-04-23 09:33:40,964] root - INFO - Inserting file: [clean_df.csv] into DB
[2023-04-23 09:33:40,964] root - INFO - Inserting data into database:[Credit_score_DB] in collection: [Credit_score_data]
[2023-04-23 09:33:43,352] root - INFO - Insertion into DB is successful!!! 
[2023-04-23 09:33:43,352] root - INFO - Fetching entire data from DB
[2023-04-23 09:33:44,755] root - INFO - Data Fetched from collection: [None] successfully!!!
[2023-04-23 09:33:44,785] root - INFO - Entire data fetched successfully from DB!!!
[2023-04-23 09:33:44,785] root - INFO - Splitting Dataset into train and test
[2023-04-23 09:33:44,826] root - INFO - Inserting new Training Data into DB
[2023-04-23 09:33:44,826] root - ERROR - 
                        error occured in script: 
                        [C:\Users\Sumeet Maheshwari\Desktop\end to end project\credit_score_prediction\credit_score_classification\credit_score\pipeline\training_pipeline.py] at 
                        try block line number: [34] execption block line number: [37]
                        error message: [
                        error occured in script: 
                        [C:\Users\Sumeet Maheshwari\Desktop\end to end project\credit_score_prediction\credit_score_classification\credit_score\pipeline\training_pipeline.py] at 
                        try block line number: [26] execption block line number: [28]
                        error message: [
                        error occured in script: 
                        [C:\Users\Sumeet Maheshwari\Desktop\end to end project\credit_score_prediction\credit_score_classification\credit_score\components\data_ingestion.py] at 
                        try block line number: [141] execption block line number: [143]
                        error message: [
                        error occured in script: 
                        [C:\Users\Sumeet Maheshwari\Desktop\end to end project\credit_score_prediction\credit_score_classification\credit_score\components\data_ingestion.py] at 
                        try block line number: [105] execption block line number: [134]
                        error message: [create_and_check_collection() got an unexpected keyword argument 'coll_name']]]]
[2023-04-23 09:33:44,992] root - INFO - 
******************** Data Ingestion log completed ********************

